### DISCUSSION TITLE
Using ryo with the Blockfrost SDK (python)

### DISCUSSION THREAD

**Discussion by mrhurless** [blockfrost/blockfrost-backend-ryo#100] [Q&A]
Should this work with the blockfrost SDK?  I think I'm running into a problem when I instantiate `BlockFrostApi` with the `base_url` pointing to this, it's adding an api version "v0" to the request, so I get an error.  I couldn't see anywhere to configure this and I can't seem to override it with an `api_version` of `None` or an empty string.  

If I just use `requests` and query the `health` endpoint (without the version v0 in the url) I get the expected response:
`{'is_healthy': True}`

Any help/pointers would be appreciated!

**Comment by mmahut** [comment]
It would be awesome for SDKs to support different Blockfrost deployment, but it is up to different owners, as each SDK is a separate project.

I would recommend to open a bug in [blockfrost-python](https://github.com/blockfrost/blockfrost-python/issues) and we will do our best to get it funded for all the SDKs.
